src/shadowbox/server/server_access_key.ts

```diff
@@ -47,11 +48,7 @@ class ServerAccessKey implements AccessKey {
   public isOverDataLimit = false;
   constructor(
       readonly id: AccessKeyId, public name: string, public metricsId: AccessKeyMetricsId,
-      readonly proxyParams: ProxyParams) {}
-}
-
-function isValidAccessKeyDataLimit(limit: DataLimit): boolean {
```

Comment:
Why did you remove validation in this class? Could be useful for unit testing.

Reply:
I removed it since this was duplicating the validation done at the API boundary.  We can export the validation function from manager_service if we need for testing, but according to the layering we've set up we're already in typescript here with a known-valid data limit, so it seemed redundant to validate again.

We weren't using it for unit testing anyways.